tests/test.rs

```diff
@@ -311,3 +311,13 @@ fn range_version_full() {
         .probe("escape")
         .unwrap();
 }
+
+#[test]
+fn rpath() {
+    let _g = LOCK.lock();
+    reset();
+    let lib = find("rpath").unwrap();
+    assert!(lib
+        .ld_options
+        .contains(&vec!["-rpath".to_string(), "/usr/local/lib".to_string(),]));
```

Comment:
This seems potentially problematic. If you split them all by comma and put them into a single `Vec`, how would you know which belong together into the same argument?

Reply:
`ld_options` is`` a Vec<Vec<String>>. The outer Vec is for the different -Wl, invocations and the inner Vec is split on the commas.

Something tells me you are mixing up contains with assert_eq.  But I could be wrong.

Reply:
Yes sorry, I was misreading that line indeed :)